RNPublisherBanner.js

```diff
@@ -98,6 +99,11 @@ PublisherBanner.propTypes = {
    */
   adUnitID: string,
 
+  /**
+   * DFP ad unit ID
```

Comment:
this comment correct?

Reply:
These are not the comments you are looking for. fixed